scripts/check_blocking_packages.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+import argparse
+
+import rosdistro
+import rosdistro.dependency_walker
+
+index = rosdistro.get_index(rosdistro.get_index_url())
+valid_distro_keys = index.distributions.keys()
+valid_distro_keys.sort()
```

Comment:
This block should happen after `argparse`. So if the arguments are invalid the error is immediate and not after fetching remote data. It also would allow to add a index url argument later.